packages/contracts/contracts/optimistic-ethereum/OVM/accounts/OVM_ECDSAContractAccount.sol

```diff
@@ -69,6 +71,38 @@ contract OVM_ECDSAContractAccount is iOVM_ECDSAContractAccount {
             Lib_ExecutionManagerWrapper.ovmCHAINID()
         );
 
+        // Compute the intrinsic gas
+        uint256 intrinsicGas = SafeMath.mul(
```

Comment:
Are we sure this calculation is correct? If not, what do we need to do to be sure?

Reply:
I think best to make a helper function with a clear explanation for the calculation and why it's correct.